### DISCUSSION TITLE
419 PAGE EXPIRED CSRF Token Error in laravel backpack

### DISCUSSION THREAD

**Discussion by Pramesh1998** [Laravel-Backpack/community-forum#674] [Q&A (Help)] [answered]
Hey i am using laravel backpack pro version and i am facing 419 page expired csrf token issue . the issue is in the search box and  delete functions. What to do to solve this problem. currently i am using with keeping my all search link in verifycsrftoken page but i dont think that best way.

**Comment by pxpm** [comment, marked as answer]
Hey @Pramesh1998 

Do a `php artisan basset:check`. It will probably tell you where your error is. 

My guess would be be `APP_URL` in `.env` file.

Let me know.
Cheers

**Reply by rajunlmtd** [reply]
It worked, thanks.

Cheers.
